quickget

```diff
@@ -55,6 +55,7 @@ function pretty_name() {
     popos)              PRETTY_NAME="Pop!_OS";;
     regolith)           PRETTY_NAME="Regolith Linux";;
     rockylinux)         PRETTY_NAME="Rocky Linux";;
+    tails)              PRETTY_NAME="Tails";;
```

Comment:
This line is not needed because it is done by the default case.

Reply:
You're right, I didn't see that. I removed that line.